assets/airlines/uae.json

```diff
@@ -13,34 +13,29 @@
     },
     "fleets": {
         "default": [
-            ["A319", 1],
-            ["A388", 115],
-            ["B77L", 21],
-            ["B77W", 134]
+            ["A388", 120],
+            ["B77L", 10],
+            ["B77W", 124]
         ],
         "cargo": [
-            ["B77L", 11]
+            ["B77X", 10]
```

Comment:
The B77X isn't in service yet 😉 Furthermore, it's not defined in the sim, so this would throw errors